api/server/router/build/build_routes.go

```diff
@@ -92,6 +50,7 @@ func newImageBuildOptions(ctx context.Context, r *http.Request) (*types.ImageBui
 	options.CPUSetCPUs = r.FormValue("cpusetcpus")
 	options.CPUSetMems = r.FormValue("cpusetmems")
 	options.CgroupParent = r.FormValue("cgroupparent")
+	options.Tags = r.Form["t"]
```

Comment:
@anusha-ragunathan Windows CI was just recently enabled, and many tests are currently skipped on master. I have a series of PRs in progress to enable the Builder tests on master.

Once they are merged Windows CI will be automatic, but I just ran all my in progress work locally for now and everything is good :+1: